src/codemagic/apple/resources/resource.py

```diff
@@ -34,34 +34,12 @@ class ResourceId(str):
     pass
 
 
-class DictSerializable:
-    _OMIT_KEYS: Tuple[str, ...] = ("_raw",)
-    _OMIT_IF_NONE_KEYS: Tuple[str, ...] = tuple()
-
+class AppleDictSerializable(DictSerializable):
```

Comment:
`DictSerializable` is now defined in `codemagic.models`. It is identical to what is here apart from how `datetime` objects are serialized.